charts/kubernetes-agent/templates/tentacle-deployment.yaml

```diff
@@ -105,6 +105,8 @@ spec:
             {{- end }}
             - name: "OCTOPUS__TENTACLE__LOGLEVEL"
               value: {{ .Values.agent.logLevel }}
+            - name: "OCTOPUS__K8STENTACLE__KUBERNETESMONITORENABLED"
+              value: {{ .Values.kubernetesMonitorChart.enabled | quote }}
```

Comment:
You can reference values with hyphens in the name directly here, the guidance is not to have hyphens in the name: 

https://helm.sh/docs/chart_best_practices/values/#naming-conventions